themes/default/content/blog/troubleshooting-builds-with-diagnostics/index.md

```diff
@@ -0,0 +1,154 @@
+---
+title: "Troubleshooting Builds With Diagnostics"
```

Comment:
Why “builds”?  From the content of the article, I take it this is about Pulumi deployments?

Reply:
Old language that I hadn't trimmed out, really: Infrastructure gets built vs applications get deployed to that infra. Good callout; I'll fix it.